<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnewFile"/>
   <addaction name="actionopenFile"/>
   <addaction name="actionsaveFile"/>
   <addaction name="actionsaveAsFile"/>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionundo"/>
   <addaction name="actioncopy"/>
   <addaction name="actionpaste"/>
   <addaction name="actioncut"/>
   <addaction name="actionselectAll"/>
  </widget>
  <action name="actionnewFile">
   <property name="text">
    <string>newFile</string>
   </property>
  </action>
  <action name="actionopenFile">
   <property name="text">
    <string>openFile</string>
   </property>
  </action>
  <action name="actionsaveFile">
   <property name="text">
    <string>saveFile</string>
   </property>
  </action>
  <action name="actionsaveAsFile">
   <property name="text">
    <string>saveAsFile</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>paste</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="text">
    <string>cut</string>
   </property>
  </action>
  <action name="actionselectAll">
   <property name="text">
    <string>selectAll</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
